app/view_models/workarea/storefront/search_autocomplete_view_model.rb

```diff
@@ -15,15 +24,19 @@ def response
       end
 
       def searches
-        @searches ||= Metrics::SearchByWeek.autocomplete(
+        return results unless results.blank?
+
+        Workarea::Insights::TrendingSearches.current.results.map do |v|
```

Comment:
This should also be memoized, since otherwise it will do the query more than once.